web/modules/custom/usagov_ssg_postprocessing/src/Controller/ssgStatController.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Drupal\usagov_ssg_postprocessing\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+
+/**
+ * An usagov_ssg_postprocessing controller for stating the current static-site-gen status.
+ */
+class ssgStatController extends ControllerBase {
+
+    public function content() {
+
+        $date = \Drupal::state()->get('ssg_stat_date');
+        $msg = \Drupal::state()->get('ssg_stat_msg');
+
+        if ( empty($msg) ) {
+            $markup = "Static Site Generator has not been run on this environment yet.";
+        } else {
+            date_default_timezone_set('America/New_York');
```

Comment:
I should have thought of this earlier, but .. as long as we're displaying a Drupal page, we don't need to set the timezone. It's probably nicer to use the default that's already set according to the user's profile.

Reply:
Ok, this has been fixed